test/fixtures/uglify.js

```diff
@@ -307,7 +307,7 @@ if(!String.prototype.trim) {
 function definePropertyWorks() {
   try {
     Object.defineProperty({}, "property", {});
-    return "property" in object;
+    return true;
```

Comment:
I don't mind this change, but out of curiosity, is this is difference in JSC? Why did you have to change it?

Reply:
Bun complains that the variable `object` doesn't exist, which if you read the code, it doesn't. I'm not sure how this original code works in other runtimes but clearly someone already didn't like this implementation as the compatibility shim was changed upstream.

Reply:
Hum, good point, this is clearly a mistake.